rootfs/conf/kops/helmfile.yaml

```diff
@@ -122,208 +127,118 @@ releases:
 
     ### Required: CHARTMUSEUM_HOSTNAME; e.g. charts.us-west-2.staging.cloudposse.org
     - name: 'ingress.hosts.{{ env "CHARTMUSEUM_HOSTNAME" | replace "." "\\." }}[0]'
-      value: "/"
+      value: "/charts"
+
+    ### Required: CHARTMUSEUM_HOSTNAME; e.g. charts.us-west-2.staging.cloudposse.org
+    - name: 'ingress.hosts.{{ env "CHARTMUSEUM_HOSTNAME" | replace "." "\\." }}[1]'
+      value: "/index.yaml"
 
     ### Optional: CHARTMUSEUM_SECRET_NAME; e.g. chartmuseum-server-tls
     - name: "ingress.tls[0].secretName"
-      value: '{{ coalesce (env "CHARTMUSEUM_SECRET_NAME") "chartmuseum-server-tls" }}'
+      value: '{{ env "CHARTMUSEUM_SECRET_NAME" | default "chartmuseum-server-tls" }}'
 
     ### Required: CHARTMUSEUM_HOSTNAME; e.g. charts.us-west-2.staging.cloudposse.org
     - name: "ingress.tls[0].hosts[0]"
       value: '{{ env "CHARTMUSEUM_HOSTNAME" }}'
 
 
 ################################################################################
-## Chart Repo ##################################################################
+## Chart Museum API ############################################################
 ################################################################################
 
-# References:
-#   - https://github.com/cloudposse/charts/blob/master/incubator/chart-repo/values.yaml
-#   - https://github.com/kubernetes/charts/blob/master/stable/chartmuseum/values.yaml
-#
-- name: "charts"
+- name: "charts-api"
   namespace: "kube-system"
   labels:
-    chart: "chart-repo"
+    chart: "chartmuseum-api"
     component: "platform"
     namespace: "kube-system"
     vendor: "kubernetes-helm"
-    default: "false"
-  chart: "cloudposse-incubator/chart-repo"
-  version: "0.2.1"
+    default: "true"
+  chart: "stable/chartmuseum"
+  version: "1.4.0"
   values:
-    - "values/chart-repo.yaml"
```

Comment:
was `chart-repo.yaml` renamed to `chartmuseum.yaml` in the repo?

Reply:
No. There is no need in it.